hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/processor/SourceProcessors.java

```diff
@@ -180,14 +182,14 @@ public static ProcessorSupplier readRemoteMapP(@Nonnull String mapName, @Nonnull
     @Nonnull
     public static <T, K, V> ProcessorMetaSupplier streamRemoteMapP(
             @Nonnull String mapName,
-            @Nonnull ClientConfig clientConfig,
+            @Nonnull String clientXml,
```

Comment:
We force the user to pass client XML as a string, but the conversion to string isn't public API. Also we change this overload of `streamRemoteMapP`, but not the other overload. I think we should stick with the original signature and convert to string for the user in the method body.

Reply:
I understand the problem and I am aware that we need to hide this internal usage inside our code but I couldn't manage to do it. When I move this conversion to one step deeper, I start having trouble serializing the Pipeline. I have not enough knowledge about the serialization of nested lambdas to solve this problem. Where should I look for reference?